ext/config/public/services.d/bootstrap.cfg

```diff
@@ -0,0 +1,5 @@
+# To enable the CA service, leave the following line uncommented
+puppetlabs.services.ca.certificate-authority-service/certificate-authority-service
+# To disable the CA service, comment out the above line and uncomment the line below
+#puppetlabs.services.ca.certificate-authority-disabled-service/certificate-authority-disabled-service
+
```

Comment:
I just noticed this file exists. It's not supposed to be in the PR. There is a ca.cfg below in the ezbake/config/ dir. Must have borked a copy/paste somewhere